plugins/inputs/redis/redis_test.go

```diff
@@ -7,11 +7,27 @@ import (
 	"testing"
 	"time"
 
+	"github.com/go-redis/redis"
 	"github.com/influxdata/telegraf/testutil"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+type testClient struct {
+}
+
+func (t *testClient) BaseTags() map[string]string {
+	return map[string]string{"host": "redis.net"}
```

Comment:
why is this hardcoded?

Reply:
Oh just for testing purposes, i am testing the logic around external calls. The struct needs this method to adhere to the interface

Reply:
ah, it's just a test. never mind.